tools/development/nnstreamerCodeGenCustomFilter.py

```diff
@@ -121,13 +121,14 @@ def get_input(text):
   in_info->info[0].dimension[0] = 3; /** @todo MODIFY THIS! */
   in_info->info[0].dimension[1] = 224; /** @todo MODIFY THIS! */
   in_info->info[0].dimension[2] = 224; /** @todo MODIFY THIS! */
+  in_info->info[0].dimension[3] = 1; /** @todo MODIFY THIS! */
 
   /**
    * Check max dimension (NNS_TENSOR_RANK_LIMIT) and
-   * fill 1 to uninitialized dimension values.
+   * fill 0 to uninitialized dimension values.
    */
-  for (i = 3; i < NNS_TENSOR_RANK_LIMIT; i++)
-    in_info->info[0].dimension[i] = 1;
+  for (i = 4; i < NNS_TENSOR_RANK_LIMIT; i++)
+    in_info->info[0].dimension[i] = 0;
 
```

Comment:
Is there any different between nnstreamerCodeGenCustomFilter.py 's `getInputDim` and cppfilter_test.cc's `getInputDim`?
It's just a question cause there is 0 init only in nnstreamerCodeGenCustomFilter.py

Reply:
Before calling the callback getinputdim(), tensor-filter initializes tensors-info struct. This 0-init code is just for clearly defining the tensor dimension.